### PR TITLE
add pt_br locale to gregorian calendar

### PR DESCRIPTION
**Pull request by guilherme-luccas**
Hi, 
I did this PR because I'm a brazilian and really liked your lib but doesn't have the option to portuguese from Brazil.
thanks

### CODE REVIEW

File: test/date_object.test.js [hunk 1]
```diff
@@ -138,8 +139,8 @@ describe("new instance", () => {
   });
 
   test("from number (21 August 2020)", () => {
-    const date = new Date(1597994736000);
-    const dateObject = new DateObject(1597994736000);
+    const date = new Date(1598021736000);
```
Comment (shahabyazdi) [inline test/date_object.test.js:142]: This timestamp (1598021736000) refers to 8/21/2020, 7:25:36 PM. But the previous timestamp (1597994736000) referred to 8/21/2020, 11:55:36 AM. This change caused the test to fail

Comment (guilherme-luccas) [reply test/date_object.test.js:142]: for me it was the opposite, when I ran the run test for the first time it accused me of an error right in that test, so I changed it

Comment (shahabyazdi) [reply test/date_object.test.js:142]: I will check it again

Comment (guilherme-luccas) [reply test/date_object.test.js:142]: image

**Pull request merged**